server/src/main/java/io/deephaven/server/plugin/python/ObjectTypeAdapter.java

```diff
@@ -34,11 +34,46 @@ public boolean isType(Object object) {
     }
 
     @Override
-    public void writeCompatibleObjectTo(Exporter exporter, Object object, OutputStream out) throws IOException {
-        final byte[] bytes = objectTypeAdapter.call(byte[].class, "to_bytes",
-                ExporterAdapter.class, new ExporterAdapter(exporter),
-                PyObject.class, (PyObject) object);
-        out.write(bytes);
+    public MessageStream compatibleClientConnection(Object object, MessageStream connection) {
+        if (objectTypeAdapter.hasAttribute("create_client_connection")) {
```

Comment:
As written, this doesn't work, there is an(other) adapter written in python that implements all methods, so we can't test for it in this way. The adapter will need to implement this instead.